### ISSUE TITLE
Problem with donttest

### ISSUE THREAD

**Issue by kjeldbeg** [SystemRDL/systemrdl-compiler#167]
Maybe this is related to https://github.com/SystemRDL/systemrdl-compiler/issues/27.

This example fails (stack trace) with 1.25.5:

```
addrmap myMap {
  reg {
    donttest;
    field {
    } myField;
  } myReg;
};

```

it works if I use:

`donttest = true;`

**Comment by amykyta3** [comment]
Thanks!
Looks like this was an edge case that was undiscovered for a while. Fixed in v1.25.6

**Comment by kjeldbeg** [comment]
You're the best!
